### ISSUE TITLE
is more complex filtering possible?

### ISSUE THREAD

**Issue by sebdalgarno** [poissonconsulting/fwapgr#19]
with pg_featureserv

e.g. a filter of list(gnis_name = "Sangan River") is acceptable

but how to get all stream_order > 1 
or how to get multiple values from the same column (property)?

**Comment by dr-jts** [comment]
Currently there is no support in `pg_featureserv` for complex filtering (such as comparison operators or multiple property values).  We are looking in to this as a medium-term enhancement.  Details are [here](https://github.com/CrunchyData/pg_featureserv/issues/28).

**Comment by sebdalgarno** [comment]
thanks @dr-jts that would be very useful! I'll monitor this closely.

**Comment by joethorley** [comment]
@sebdalgarno Yes this would be really helpful as I would like to transfer the following code for `fwa_feature()` to `fwa_collection()`.

```
fwa_feature("fwa_rivers_poly", filter = "gnis_name_1 IN ('Lardeau River', 'Duncan River')") 
```

**Comment by sebdalgarno** [comment]
@joethorley for now this will have be done with a loop or apply function, e.g. 
```
map_df(c('Lardeau River', 'Duncan River'), function(x) fwa_collection("fwa_rivers_poly", filter = list("gnis_name_1" = x)))
```
until the API is updated. This was one advantage of fwa_feature and the old API but I dont think it's worth keeping the old API around just for that

**Comment by sebdalgarno** [comment]
see latest update from CrunchyData [here](https://github.com/CrunchyData/pg_featureserv/issues/69)

**Comment by joethorley** [comment]
interesting - what could we use it for?
